cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ModelUtils.java

```diff
@@ -44,6 +55,7 @@
 import java.util.List;
 import java.util.Map;
 
+@SuppressWarnings("checkstyle:ClassFanOutComplexity")
```

Comment:
Hmm, I  know the original issue said `ModelUtils` But I wonder if we should create some separate `StorageUtils` or `VolumeUtils` class or something instead of just moving the methods from one gigantic class to another one.

Reply:
@scholzj Good idea. I will split the ModelUtils into another class.